sema/bench/core.py

```
@@ -10,12 +10,24 @@
 
+from sema.commons.service import ServiceBase, ServiceResult, Trace
 from sema.bench.dispatcher import TaskDispatcher
 from sema.bench.task import Task
 
 log = getLogger(__name__)
 
 
+class SembenchResult(ServiceResult):
+    """Result of the sembench service"""
+
+    def __init__(self):
```

Ruff

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)